### ISSUE TITLE
Change the name display

### ISSUE THREAD

**Issue by DaPoHou** [fisharebest/webtrees#3565]
On the individual page, the name is currently displayed with a double slash "/"
I would like to change the name display back to the original way.

![微信图片_20201021152637](https://user-images.githubusercontent.com/26761596/96686963-d2b2bb80-13b1-11eb-883e-834e76bcdf58.png)

**Comment by JustCarmen** [comment]
I totally agree. It is really ugly and I don't see the benefit of this change.

**Comment by fisharebest** [comment]
> I don't see the benefit of this change.

Previously, this showed the exact same name as appears in the header.

By showing the raw data, you can see which part of the name is a surname (or if no surname exists).

e.g.

```
1 NAME Jan Peterson
1 NAME Jan /Peterson/
```

Remember that the GIVN/SURN fields may not be present.
